.gitignore

```diff
@@ -9,3 +9,6 @@ __pycache__
 
 # stubgen
 out
+
+# intellij
+.idea
```

Comment:
I prefer for ppl to use their own global `.gitignore` on IDE specific files.

```suggestion
```

Reply:
What's the harm? Does .gitignore need to be clean?

Reply:
I wouldn't accept this on the projects I maintain cause it opens space for ppl to keep adding stuff. I prefer correctness. The gitignore for a project should only contain what is generated from the project itself but shouldn't be committed.

Reply:
removed